CDCgov/prime-simplereport
# Bob/4358 obfuscate token in logs

# FRONTEND PULL REQUEST
Adds back in [4358](https://github.com/CDCgov/prime-simplereport/pull/5486) but with regex that doesn't break safari.

## Changes Proposed

- Replaces the regex replacement with one that is accepted by Safari. Extra changes on top of the changes from last week can be seen [here](https://github.com/CDCgov/prime-simplereport/pull/5528/files/ce7b340f8542dd85e17c816527e298930f303be7..077a72de48dc3bc996fab3d8fef572ca9148ff2b7)
 
## Testing
deployed on dev6

- Log in on dev6 to trigger application insights event
- Go to the logs and search for the [home page event blade](https://portal.azure.com#@9ce70869-60db-44fd-abe8-d2767077fc8f/blade/Microsoft_OperationsManagementSuite_Workspace/Logs.ReactView/resourceId/%2Fsubscriptions%2F7d1e3999-6577-4cd5-b296-f518e5c8e677%2FresourceGroups%2Fprime-simple-report-dev%2Fproviders%2Fmicrosoft.insights%2Fcomponents%2Fprime-simple-report-dev6-insights/source/LogsBlade.AnalyticsShareLinkToQuery/q/H4sIAAAAAAAAAytITE8Ny0wtL%252BaqUcgvSkktUkiqVCjJzE0tLknMLVBISS1OBgBvA%252BPnIwAAAA%253D%253D/timespan/PT30M)
Notice that the id token for the URI redirect is obscured
- Double check that this works on Safari too.

## Screenshots / Demos

- For large changes, please pair with a designer to ensure changes are as intended
